test/functionalTest/cases/1308_subscription_cache/subCacheMixedSubscriptionsWithUpdatesAndDeletes.test

```diff
@@ -0,0 +1,578 @@
+# Copyright 2015 Telefonica InvesTigacion y Desarrollo, S.A.U
```

Comment:
I understand that this new test in this merge was already reviewed in a previos PR in the 0.24.0 sub-branches. Is that correct? Just to save some reviewing time... ;)